Composer/packages/client/src/pages/publish/type.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { DialogSetting, PublishResult, PublishTarget } from '@bfc/shared';
+
+import { PublishType } from '../../recoilModel/types';
+
+export type IBotStatus = {
+  id: string;
+  name: string;
+  publishTargets?: PublishTarget[];
+  publishTarget?: string;
+  time?: string;
```

Comment:
timestamp?

Reply:
Using `time` is for sync with `publishResult` type